### PR TITLE
Fix(troubleshooting sections 2nd batch): add icons and standardized sections

### PR DESCRIPTION
**Pull request by NathalieC**
* in Bonita Studio pref and config: add image for troubleshooting
* in Bonita Studio pref and config: add line breaks in troubleshooting section

### CODE REVIEW

File: modules/identity/pages/single-sign-on-with-oidc.adoc [hunk 1]
```diff
@@ -286,7 +286,9 @@ To setup Bonita for OIDC logout:
 If the logout flow supported by your OIDC provider is not the same as the one supported by Bonita platform, the preferred solution to handle it anyway is to intercept the requests to /logoutService and handle the logout programmatically.
 ====
 
-== Troubleshoot
```
Comment (alachambre) [inline modules/identity/pages/single-sign-on-with-oidc.adoc:289]: Why is this section not at the bottom of the page, like in all other pages?

File: modules/identity/pages/single-sign-on-with-saml.adoc [hunk 2]
```diff
@@ -339,7 +339,9 @@ To setup Bonita for global logout:
 If the single logout flow supported by your IdP is not the same as the one supported by Bonita platform, the preferred solution to handle it anyway is to intercept the requests to /logoutService and handle the logout programmatically.
 ====
 
-== image:images/troubleshooting.png[troubleshooting-icon] Troubleshooting
+== image:ROOT:images/troubleshooting.png[troubleshooting-icon] Troubleshooting
```
Comment (alachambre) [inline modules/identity/pages/single-sign-on-with-saml.adoc:379]: Why is this section not at the bottom of the page?

**Review by alachambre — CHANGES_REQUESTED** [review]
I think the two sso troobleshooting sections should be moved at the bottom of the pages.

File: modules/identity/pages/single-sign-on-with-oidc.adoc (hunk 1 above)
Comment (NathalieC) [reply modules/identity/pages/single-sign-on-with-oidc.adoc:289]: thanks. done.

File: modules/identity/pages/single-sign-on-with-saml.adoc (hunk 2 above)
Comment (NathalieC) [reply modules/identity/pages/single-sign-on-with-saml.adoc:379]: thx. done

**Pull request closed without merging**